scripts/build-cdn-indexes.ts

```diff
@@ -1,37 +1,99 @@
 import path from 'path';
 import themes from '../themes.json';
+import packageVersions from '../versions.json';
 import fs from 'node:fs';
 import c from 'chalk';
 
-const getSystems = () => {
-  const result: string[][] = [];
-  Object.entries(themes).forEach(([themeName, themeInfo]) => {
-    if ('incomplete' in themeInfo) return;
-
-    const packageFile = path.join('packages', themeInfo.packageName, 'package.json');
-    const version = JSON.parse(fs.readFileSync(packageFile, 'utf8')).version;
-
-    result.push([themeInfo.packageName, version, themeName]);
-  });
-  return result;
-};
-
-const codeBlock = (lines: string[]) => {
+function codeBlock(lines: string[]) {
   const escaped = lines.join('\n').replace(/</g, '&lt;').replace(/>/g, '&gt;');
   const stringToCopy = JSON.stringify(lines);
   return `
     <pre class="ds-u-fill--gray-lightest ds-u-font-size--sm ds-u-padding--1 ds-u-margin-y--1 ds-u-overflow--auto"><code>${escaped}</code></pre>
     <ds-button size="small" onclick='navigator.clipboard.writeText(${stringToCopy}.join("\\n"))'>Copy snippet</ds-button>
   `;
-};
+}
+function renderPageHtml(theme: keyof typeof themes, title: string, mainContent: string) {
+  const system = themes[theme].packageName;
+  const version = packageVersions[system as keyof typeof packageVersions][0];
+  return `<!DOCTYPE html>
+  <html lang="en">
+    <meta charset="utf-8" />
+    <head>
+      <meta name=“viewport” content=“width=device-width, initial-scale=1, shrink-to-fit=no”>
+      <title>${title} - CMSDS</title>
+      <link rel="stylesheet" href="https://design.cms.gov/cdn/${system}/${version}/css/index.css" />
+      <link rel="stylesheet" href="https://design.cms.gov/cdn/${system}/${version}/css/${theme}-theme.css" />
+    </head>
+    <body class="ds-content" style="margin: 0">
+      <ds-usa-banner></ds-usa-banner>
+      <header class="ds-base--inverse ds-u-padding-y--3">
+        <div class="ds-l-container">
+          <h1 class="ds-text-heading--2xl">${title}</h1>
+        </div>
+      </header>
+      <div class="ds-l-container ds-content ds-u-padding-y--4">
+        ${mainContent}
+      </div>
+      <script src="https://design.cms.gov/cdn/${system}/${version}/web-components/bundle/web-components.js"></script>
+    </body>
+  </html>`;
+}
+
+function writeCdnIndex() {
+  const theme = 'core';
+
+  const packageSections = Object.keys(themes).map((theme) => {
+    const { packageName } = themes[theme as keyof typeof themes];
+    const versions = packageVersions[packageName as keyof typeof packageVersions];
+
+    const renderItem = (version: string) => `
+      <li>
+        <a href='https://design.cms.gov/cdn/${packageName}/${version}'>${version}</a>
+      </li>
+    `;
+
+    return `
+      <section class="ds-u-margin-top--3 ds-u-measure--wide">
+        <h2>${packageName}</h2>
+        <ul style="column-count: 4; display: block">
```

Comment:
an a11y enhancement for this could be adding an `id` to the `<h2>` and `aria-labelledby` on the lists to give the lists a name. otherwise looks good.

Reply:
`column-count` looks a little goofy on mobile but i don't know if anyone cares about mobile when referencing CDN

Reply:
I'm having trouble figuring out why the page looks weird on mobile. I have the viewport meta tag that is supposed to fix that.

Reply:
It was that my quotes weren't real quotes on the meta tag 🤦‍♂️